app/controllers/merchant_invoices_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MerchantInvoicesController < ApplicationController
```

Comment:
I'll have to check the user stories but we might be able to just use the merchant controller for this since we're only grabbing a merchant from it. I believe we will need a merhant_invoice controller at some point though

Reply:
I could be way wrong on this too

Reply:
Scratch this- after checking the routes this is the way to go

Reply:
Sounds good, thanks for checking,